validator/client/beacon-api/duties.go

```diff
@@ -220,22 +187,18 @@ func (c *beaconApiValidatorClient) dutiesForEpoch(
 		if committeeMappingKey, ok := attesterDutiesMapping[v.index]; ok {
 			committeeIndex = committeeMappingKey.committeeIndex
 			attesterSlot = committeeMappingKey.slot
-
-			if committeeValidatorIndices, ok = committeeMapping[committeeMappingKey]; !ok {
-				return nil, errors.Errorf("failed to find validators for committee index `%d` and slot `%d`", committeeIndex, attesterSlot)
-			}
+			committeeValidatorIndices = make([]primitives.ValidatorIndex, committeeMappingKey.committeeLength) // make a fake committee because only length is used
 		}
 
 		duties[i] = &ethpb.DutiesResponse_Duty{
-			Committee:        committeeValidatorIndices,
-			CommitteeIndex:   committeeIndex,
-			AttesterSlot:     attesterSlot,
-			ProposerSlots:    proposerDutySlots[v.index],
-			PublicKey:        v.pubkey,
-			Status:           v.status,
-			ValidatorIndex:   v.index,
-			IsSyncCommittee:  syncDutiesMapping[v.index],
-			CommitteesAtSlot: slotCommittees[strconv.FormatUint(uint64(attesterSlot), 10)],
```

Comment:
we don't use this in gRPC , haven't found a usecase yet

Reply:
I can't find any either